src/hooks/useLogDownloader/index.ts

```diff
@@ -142,14 +153,19 @@ const useLogDownloader = (
           });
           setIsLoading(false);
         });
+    } else if (!url && !isLoadingEvergreen) {
+      dispatchToast.error("Log URL not specified, unable to download.", true, {
+        shouldTimeout: false,
+      });
```

Comment:
Ah ok, does it make sense to use this existing pattern in the file?

```suggestion
      setError("Log URL not specified, unable to download.");
```